frontend/src/components/elements/Alert/Alert.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license
+ * Copyright 2018-2019 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { ReactNode } from "react"
+
+import classNames from "classnames"
+import { Map as ImmutableMap } from "immutable"
+import { Alert as AlertProto } from "autogen/proto"
+import { StreamlitMarkdown } from "components/shared/StreamlitMarkdown"
+
+import "assets/css/write.scss"
+
+// classes defined in assets/css/theme.scss
+const ALERT_CSS_CLASS = ImmutableMap({
+  [AlertProto.Format.SUCCESS]: "alert-success",
+  [AlertProto.Format.INFO]: "alert-info",
+  [AlertProto.Format.WARNING]: "alert-warning",
+  [AlertProto.Format.ERROR]: "alert-danger",
+})
```

Comment:
Nit: why not just use a plain JS object here?

```ts
const ALERT_CSS_CLASS = {
  [AlertProto.Format.SUCCESS]: "alert-success",
  [AlertProto.Format.INFO]: "alert-info",
  [AlertProto.Format.WARNING]: "alert-warning",
  [AlertProto.Format.ERROR]: "alert-danger",
}
```

...it's simpler and means you don't need to `toString()` later.

Reply:
Yeah, that's what I had wanted as well, but it doesn't compile:   

`Element implicitly has an 'any' type because expression of type 'any' can't be used to index type '{ [AlertProto.Format.SUCCESS]: string; [AlertProto.Format.INFO]: string; [AlertProto.Format.WARNING]: string; [AlertProto.Format.ERROR]: string; }'.  TS7053`

Reply:
I spent too long of a time trying to make the simple thing work, and it seems it's just too mushy for Typescript.